test/server/data/test-suites/fixture-is-missing/testfile.js

```diff
@@ -0,0 +1,6 @@
+// fixture `Fixture`
```

Comment:
Should we cover related scenarios?

```js
// fixture `Fixture`

//test('Test', () => {});
```


```js
fixture `Fixture`

//test('Test', () => {});
```

Reply:
There will be another error in these cases: NO TESTS to run

Reply:
Ok